### PR TITLE
Feat: Add culture context

### LINKED ISSUE #464: Add missing Protocol Properties
See 
- [x] https://github.com/getsentry/develop/pull/335 Already in spec but not documented
- [x] https://github.com/getsentry/develop/pull/330 Theme can also be worked on. It's for dark vs light mode.
- [ ] https://github.com/getsentry/develop/pull/329 `Culture` context.

### PR DESCRIPTION
**Pull request by ueman**
## :bulb: Motivation and Context
https://develop.sentry.dev/sdk/event-payloads/contexts/
Closes https://github.com/getsentry/sentry-dart/issues/464
Unblocks https://github.com/getsentry/sentry-dart/pull/452

## :green_heart: How did you test it?
I've written new tests

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] I updated the docs if needed
- [ ] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+* Feat: Add Culture Context
```
Comment (ueman) [inline CHANGELOG.md]:
```suggestion
* Feat: Add Culture Context (#491)
```

File: dart/lib/src/protocol/sentry_culture.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import 'package:meta/meta.dart';
+
+/// App context describes the culture.
```
Comment (ueman) [inline dart/lib/src/protocol/sentry_culture.dart]:
```suggestion
/// Culture Context describes certain properties of the culture in which the  
/// software is used.
```

File: dart/lib/src/protocol/sentry_culture.dart [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+  /// Optional. boolean, either true or false.
+  final bool? is24HourFormat;
+
+  /// Optional. The timezone of the locale. For example, `Europe/Vienna`.
+  final String? timezone;
+
+  /// Produces a [Map] that can be serialized to JSON.
+  Map<String, dynamic> toJson() {
+    return <String, String>{};
```
Comment (marandaneto) [inline dart/lib/src/protocol/sentry_culture.dart]: does that work? we should populate the map right

Comment (ueman) [reply dart/lib/src/protocol/sentry_culture.dart]: Oh shoot, yeah you're right.

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**